python/ray/actor.py

```diff
@@ -538,6 +538,9 @@ class _ActorClassMetadata:
             task.
         memory: The heap memory quota for this actor.
         resources: The default resources required by the actor creation task.
+        label_selector: The labels required for the node on which this actor
+            runs. Label selectors consist of a key-value pair, where the key
+            is a label name and the value is a label value with a condition.
```

Comment:
```suggestion
        label_selector: The labels required for the node on which this actor
            can be scheduled on. The label selector consist of key-value pairs, where the keys
            are label names and the value are expressions consisting of an operator with label values or just a value to indicate equality..
```

I think it will also be better to add the supported operators and some examples of the label expressions